pkg/envoy/ads/debugger_test.go

```diff
@@ -0,0 +1,30 @@
+package ads
+
+import (
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	tassert "github.com/stretchr/testify/assert"
+	testclient "k8s.io/client-go/kubernetes/fake"
+
+	"github.com/openservicemesh/osm/pkg/catalog"
+	"github.com/openservicemesh/osm/pkg/certificate"
+	"github.com/openservicemesh/osm/pkg/configurator"
+	"github.com/openservicemesh/osm/pkg/envoy"
+	"github.com/openservicemesh/osm/pkg/tests"
+)
+
+func TestGetXDSLog(t *testing.T) {
+	assert := tassert.New(t)
+	kubeClient := testclient.NewSimpleClientset()
+	mc := catalog.NewFakeMeshCatalog(kubeClient)
+	mockCtrl := gomock.NewController(t)
+	mockConfigurator := configurator.NewMockConfigurator(mockCtrl)
+	mockCertManager := certificate.NewMockManager(mockCtrl)
+	s := NewADSServer(mc, true, tests.Namespace, mockConfigurator, mockCertManager)
```

Comment:
small nit: we're relying on `NewADSServer` to create an empty map here, so if it changes then this unit test will break. We wouldn't want the unit test for `GetXDSLog` to break because `NewADSServer` changes. Therefore, it would be more appropriate to create a `Server` which a specific value for  the`xdsLog` attribute and then set your expectation to that value after calling the function. Lmk what you think

Reply:
I agree, I'll go ahead and make the changes